packages/dds/sequence/src/test/intervalTestUtils.ts

```diff
@@ -107,6 +108,25 @@ export function assertEquivalentSharedStrings(a: SharedString, b: SharedString)
 	}
 }
 
+function assertPropertiesEqual(a: SharedString, b: SharedString): void {
+	for (let i = 0; i < a.getLength(); i++) {
+		const aProps = a.getPropertiesAtPosition(i) ?? [];
+		const bProps = b.getPropertiesAtPosition(i) ?? [];
+		const aKeys = aProps === undefined ? [] : Object.keys(aProps);
```

Comment:
I think just doing `assert.deepEqual(aProps, bProps)` will produce a better error message than going through them key-by-key. Simpler code too :)

Reply:
Doing this gives me a failure in the afterEach hook. I thought it had something to do with the undefined/null properties, but I get the same failure even without those as potential values. The same seeds fail with the key-by-key comparison as the deepEqual comparison, so my guess is that it's the same issue, just failing in a different place. I wasn't sure if I should still change the assert given the error message, though.

Reply:
deep equal also won't work once we have handles, so we'll need to check key by key anyway